src/main/java/by/itech/lab/supplier/domain/User.java

```diff
@@ -1,199 +1,85 @@
     @Enumerated(EnumType.STRING)
-    public Role getRole() {
-        return role;
-    }
-
-    public void setRole(Role role) {
-        this.role = role;
-    }
+    private Role role;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "address_id")
-    public Address getAddress() {
-        return address;
-    }
-
-    public void setAddress(Address address) {
-        this.address = address;
-    }
+    private Address address;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "customer_id")
-    public Customer getCustomer() {
-        return customer;
-    }
-
-    public void setCustomer(Customer customer) {
-        this.customer = customer;
-    }
+    private Customer customer;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "warehouse_id")
-    public Warehouse getWarehouse() {
-        return warehouse;
-    }
-
-    public void setWarehouse(Warehouse warehouse) {
-        this.warehouse = warehouse;
-    }
+    private Warehouse warehouse;
 
     @OneToMany(mappedBy = "createdByUsers")
-    public Set<WayBill> getCreatorWayBills() {
-        return creatorWayBills;
-    }
-
-    public void setCreatorWayBills(Set<WayBill> creatorWayBills) {
-        this.creatorWayBills = creatorWayBills;
-    }
+    private Set<WayBill> creatorWayBills = new HashSet<>();
 
     @OneToMany(mappedBy = "updatedByUsers")
-    public Set<WayBill> getUpdatorWayBills() {
-        return updatorWayBills;
-    }
-
-    public void setUpdatorWayBills(Set<WayBill> updatorWayBills) {
-        this.updatorWayBills = updatorWayBills;
-    }
+    private Set<WayBill> updatorWayBills = new HashSet<>();
 
     @OneToMany(mappedBy = "driver")
-    public Set<WayBill> getDriverWayBills() {
-        return driverWayBills;
-    }
-
-    public void setDriverWayBills(Set<WayBill> driverWayBills) {
-        this.driverWayBills = driverWayBills;
-    }
-
+    private Set<WayBill> driverWayBills = new HashSet<>();
 
     @OneToMany(mappedBy = "createdByUsers")
-    public Set<Application> getCreatorApplications() {
-        return creatorApplications;
-    }
-
-    public void setCreatorApplications(Set<Application> creatorApplications) {
-        this.creatorApplications = creatorApplications;
-    }
+    private Set<Application> creatorApplications = new HashSet<>();
 
     @OneToMany(mappedBy = "lastUpdatedByUsers")
-    public Set<Application> getUpdatorApplications() {
-        return updatorApplications;
-    }
-
-    public void setUpdatorApplications(Set<Application> updatorApplications) {
-        this.updatorApplications = updatorApplications;
-    }
-
-    @Column(name = "is_active", nullable = false)
-    public boolean isActive() {
-        return active;
-    }
-
-    public void setActive(boolean active) {
-        this.active = active;
-    }
+    private Set<Application> updatorApplications = new HashSet<>();
+
+    @Column(name = "is_active")
```

Comment:
better to use simple names, like "active", so you dont need in some special mappings